ops/production-deploy.tmpl.yaml

```
@@ -228,6 +238,17 @@ worker:
   extraVolumeMounts: *volMounts
   extraEnvVars: *envVars
 
+affinity:
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+          - key: name
+            operator: In
+            values:
+              - {{ include "hyrax.fullname" . }}
+        topologyKey: "kubernetes.io/hostname"
```

Not sure if this would break anything, but this whole block looks like it's indented 2 spaces too far:

```suggestion
affinity:
  podAntiAffinity:
    requiredDuringSchedulingIgnoredDuringExecution:
    - labelSelector:
        matchExpressions:
        - key: name
          operator: In
          values:
            - {{ include "hyrax.fullname" . }}
      topologyKey: "kubernetes.io/hostname"
```

---

Thank you for this, it def needs to be moved over! Good eye!